README.md

```diff
@@ -1,34 +1,41 @@
 - Install Docker.
 - If you use Linux, you also need to install Docker Compose.
 
 Complete the installation:
 1. Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
-2. Open the **Codewind Explorer** view.
+2. Open the **Codewind Explorer** view. Go to **Window**>**Show View**>**Other…**>**Codewind**>**Codewind Explorer**.
 3. Double-click the **Codewind** entry in the view to finish installing Codewind. The download is approximately 1 GB. For more information, see [Installing Codewind for Eclipse](https://www.eclipse.org/codewind/mdteclipseinstallinfo.html).
 
 ## Using Codewind for Eclipse
 Right-click the **Local Projects** entry in the view to create new projects or add existing projects to Codewind. After a project is created or added, it displays in the **Codewind Explorer** view. Right-click the project to see the available actions.
 
-Features:</br>
-- Create new projects from application templates or add existing container-ready projects to Codewind.
-- View Codewind projects, including application and build statuses.
-- Debug **Microprofile/Java EE** and **Spring** projects in their containers.
-- Set up a Chrome debug session for **Node.js** projects.
-- View application and build logs in the **Console** view.
-- View and edit project deployment information.
-- Open a shell session into a Codewind application container.
-- Toggle project auto build and manually initiate project builds.
-- Integrate Codewind validation errors into the **Markers** view.
-- Disable, enable, and remove projects.
+[Features:](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html)</br>
+- **Open Application**: Opens the application in the default Eclipse browser. This action is only available when the application is running or debugging.
+- **Open Project Overview**: Opens the overview page for a project. You can use this action to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
+- **Open Container Shell**: Opens a shell into your application container. This action is available only when the container is active.
+- **Open Application Monitor**: Opens the application monitor in the default Eclipse browser. Use this action to monitor the activity and health of your application. This action is available only when the application is running or debugging.
+- **Import Project**: Imports your project into the Eclipse workspace.
+- **Show Log Files**: If log files are available, this action displays a list of log files. In the Eclipse **Console** view, click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
```

Comment:
If the user clicks Show All or one of the individual log file toggle actions then the log file will open in the Eclipse Console view.  So this is not correct: "In the Eclipse **Console** view, click a log file to open it, or click **Show All** to open all available log files."